packages/fxa-event-broker/.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
   "useTabs": false,
-  "printWidth": 80,
+  "printWidth": 100,
```

Comment:
80 was a bit narrow, and most of the FxA code seemed to hew to 100, so I upped it for TypeScript here.